client/modules/IDE/components/Editor/index.jsx

```diff
@@ -340,6 +340,8 @@ class Editor extends React.Component {
       mode = 'x-shader/x-fragment';
     } else if (fileName.match(/.+\.(vert|stl)$/i)) {
       mode = 'x-shader/x-vertex';
+    } else if (fileName.match(/.+\.mtl$/i)) {
```

Comment:
if we want .mtl files to have the 'x-shader/x-vertex' mode, I think it would be best to combine this with line 341 within the parentheses where `(vert|stl)` are (so maybe something like `(vert|stl|mtl)`), since the `x-shader/x-vertex' mode is already begin referenced there.